bokeh/charts/_charts.py

```diff
@@ -144,17 +147,28 @@ def start_plot(self, xgrid, ygrid):
         if ygrid:
             self.make_grid(1, yaxis.ticker)
 
-        # Add tools
+        # Add tools if supposed to
         if self.tools:
+            # only create pan and zoom tools if chart is not categorical...
+            if self.categorical:
+                tools_to_remove = ['pan', 'zoom']
+            else:
+                tools_to_remove = []
```

Comment:
> I also wonder if maybe _add_tools_to_plot shouldn't just check for categorical ranges in one place. Taking tools_to_remove is more general, but I can't think of any other use scenarios for it, and some day cat ranges will work with pan/zoom so then there would just be one place to fix up.

@bryevdv I added tools_to_remove to be more general as I thought that in the future it may be easier to exclude tools.. but I can't think of real use cases either now... 

> you check None in https://github.com/bokeh/bokeh/pull/1624/files#diff-3e6325a902133d8fd2e886fe13214143R338, maybe better to just make it None here too...

@damianavila That is a different case.. I check None just to set empty list as a default value to make https://github.com/bokeh/bokeh/pull/1624/files#diff-3e6325a902133d8fd2e886fe13214143R362 work without other checks